lib/elixir/lib/map.ex

```diff
@@ -599,8 +599,8 @@ defmodule Map do
   @doc """
   Updates the `key` in `map` with the given function.
 
-  If `key` is present in `map` with value `value`, `fun` is invoked with
-  argument `value` and its result is used as the new value of `key`. If `key` is
+  If `key` is present in `map` then the `existing_value` is passed to `fun` and its result is
```

Comment:
```suggestion
  If `key` is present in `map` then the existing value is passed to `fun` and its result is
```

since there is no `existing_value` argument anywhere. :)